Reasonable.md

```diff
@@ -0,0 +1,25 @@
+# Reasonable
+
+To be reasonable is to be ALL (conjuctive) of:
+* rational - to hold logically consistent beliefs and values
+* agnostic - to have no faith
```

Comment:
Could we rephrase agnosticism?

Reply:
The label is less important than the concept. Which do you want to change?

Reply:
I think the concept is fine, just the phrasing of the definition. How about a shortened the original definition by Huxley that you mentioned? 

> "Agnosticism is of the essence of science, whether ancient or modern. It simply means that a man shall not say he knows or believes that which he has no scientific grounds for professing to know or believe."

I.e. Judge truth by the evidence.

Reply:
I'm using "faith" here to mean to have unreasonable (i.e. non-Bayesian, non-empirical, non-scientific) confidence in a belief. Do you have a different understanding of faith or are you worried about excluding people of faith from the Vulcans?

Reply:
Its more the latter. Its about marketing imo, its easier to vouch for a philosophical doctrine rather than a religious one, even if certain religious conclusions can be drawn implicitly from the philosophy. As far as I understand this is what Stoicism does. ~84% of the world is religious, that's a large market share to cut.

Reply:
I agree that passing this proposal as it stands would place requirements on Vulcans that would exclude most candidates. This nicely dovetails into a discussion on the true purpose for this project. What is the vision and mission of the Vulcan Academy? I'm proposing here that the essence of being Vulcan is aspiring to be guided by reason. If the majority of the world would prefer to be guided by faith rather than reason, that is their right but they can't be Vulcans.

Reply:
I propose this be the discussion topic of our next Vulcan meeting 😃. I think we are all in favour of aspiring to be guided by reason. I'd argue not all facets of religion are unreasonable, and globally dismissing faith has the normative connotation that the opposite is true.

Reply:
Agreed, good topic for the next meeting. FWIW I don't think all facets of religion are unreasonable, charity for example is a common facet across all religions and is obviously good. But the facets predicated on faith are not reasonable by definition.

Reply:
![image](https://user-images.githubusercontent.com/434644/36359483-e1a1b25a-14e9-11e8-895f-5fff574cb188.png)